src/ec/packages/ec-component-ecl-compliance/components/ecl-compliance-media-container.html.twig

```diff
@@ -0,0 +1,56 @@
+{#
+  Parameters:
+    alt (string),
+    image (string),
+    embedded_media: (string),
+    sources: (array of objects),
+    tracks: (array of objects)
+#}
+
+{% extends "@ecl-twig/ec-component-ecl-compliance/ecl-compliance.html.twig" %}
+
+{% set message = '' %}
+{% set component = 'media-container' %}
+{% set no_embedded_media_case = embedded_media is not defined or embedded_media is empty %}
+{% set no_sources_case = sources is not defined or sources is empty or sources is not iterable %}
+{% set no_tracks_case = tracks is not defined or tracks is empty or tracks is not iterable %}
+{% set no_video_resources_case = no_sources_case and no_tracks_case %}
+{% set no_image_case = image is not defined or image is empty %}
+{% set no_alt_case = no_image_case == false and (alt is not defined or alt is empty) %}
+
+{# no media provided at all #}
+{% set not_compliant = no_image_case and no_embedded_media_case and no_video_resources_case %}
```

Comment:
It's very possible I'm not getting logic of cascading AND statements to catch possible wrong scenarios, though to be honest I'd expect OR statements combination.

Reply:
I can explain, this is a particular use case, the logic is always tailored to what the template is doing, and sometimes it might be that is the logic in the template is not so strong or straightforward, in general.
Anyway, here the thing is that only when all of these use cases are true the component is not valid, that's why, exceptionally, in this component the not_compliant is a concatenation of ANDS.